core/src/apps/cardano/README.md

```diff
@@ -385,3 +395,10 @@ a900818258203b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b70001
   }
 }
 ```
+
+### Babbage-era transaction body example
+This Plutus transaction body contains elements introduced in the Babbage era (perhaps the most intersting elements with regards to the CBOR encoding are the inline datum and the reference script). You can use one of the CBOR inspection tools listed on top of this document to examine the transaction body structure.
```

Comment:
```suggestion
This Plutus transaction body contains elements introduced in the Babbage era (perhaps the most interesting elements with regards to the CBOR encoding are the inline datum and the reference script). You can use one of the CBOR inspection tools listed on top of this document to examine the transaction body structure.
```

Reply:
Thanks for catching this! https://github.com/vacuumlabs/trezor-firmware/pull/78/commits/34f2cf4611ec15eb3565b7096a5dd690e23c5413